test/manual/definitions/apm/.net/linux/linux2-aspnetcore5.json

```diff
@@ -0,0 +1,51 @@
+{
+  "global_tags": {
+    "owning_team": "OpenSource",
+    "Environment": "development",
+    "Department": "Product",
+    "Product": "Virtuoso"
+  },
+
+  "resources": [{
+    "id": "host1",
+    "display_name": "AwsLinux2ASPNETCore",
+    "provider": "aws",
+    "type": "ec2",
+    "size": "t3.nano",

```

Comment:
Is nano a little small for everything we do on the box?

Reply:
No, I don't think it is.  I was able to run everything and it spun up in a reasonable amount of time.